airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -471,6 +585,286 @@ definitions:
 
+  deals_archived_stream:
+    primary_key:
+      - id
+    $ref: "#/definitions/stream_base"
+    name: deals_archived
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/base_requester"
+        path: /crm/v3/objects/deals
+        request_parameters:
+          associations: "['contacts', 'companies', 'line_items']"
+          archived: "true"
+          properties:
+            type: QueryProperties
+            property_list:
+              type: PropertiesFromEndpoint
+              property_field_path:
+                - name
+              retriever:
+                type: SimpleRetriever
+                requester:
+                  $ref: "#/definitions/base_requester"
+                  path: /properties/v2/deal/properties
+                decoder:
+                  type: JsonDecoder
+                record_selector:
+                  type: RecordSelector
+                  extractor:
+                    type: DpathExtractor
+                    field_path: []
+            property_chunking:
+              type: PropertyChunking
+              property_limit_type: characters
+              property_limit: 15000
+              record_merge_strategy:
+                type: GroupByKeyMergeStrategy
+                key: id
+      paginator:
+        $ref: "#/definitions/cursor_paginator"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - results
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["archivedAt"]
+            value: "{{ record.get('archivedAt') or record['createdAt'] }}"
+      - type: CustomTransformation
+        class_name: source_hubspot.components.NewtoLegacyFieldTransformation
+        field_mapping:
+          hs_date_entered_: "hs_v2_date_entered_"
+          hs_date_exited_: "hs_v2_date_exited_"
+          hs_time_in_: "hs_v2_latest_time_in_"
+      - type: DpathFlattenFields
+        field_path: ["properties"]
+        key_transformation:
+          type: KeyTransformation
+          prefix: properties_
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+      cursor_field: archivedAt
+      is_client_side_incremental: true
+    state_migrations:
+      - type: CustomStateMigration
+        class_name: source_hubspot.components.MigrateEmptyStringState
+        cursor_field: archivedAt
+    schema_loader:
+      - $ref: "#/definitions/deals_archived_schema_loader"
+      - type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/deals_archived"
+
+  forms_stream:
+    primary_key:
+      - id
+    $ref: "#/definitions/stream_base"
+    name: forms
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/base_requester"
+        path: /marketing/v3/forms
+        request_parameters:
+          formTypes:
+            "['hubspot', 'captured', 'flow']"
```

Comment:
Testing in Postman,  does this result in an error when formatted as a string list? I had to format each `formType=hubspot` as a separate parameter. For example:
`https://api.hubapi.com/marketing/v3/forms?limit=100&formTypes=captured&formTypes=blog_comment&formTypes=hubspot&formTypes=flow`